src/styles/card.css

```diff
@@ -0,0 +1,98 @@
+/* region card */
```

Comment:
Not all IDEs are supporting folding regions. Delete it

Reply:
Ok, fixed